canonical-websites/www.ubuntu.com
# Add ./run script for local development

Added a ./run script to use Docker for local development in a hopefully very simple way.

Fixes #1275.

QA
==

Follow the README.md and HACKING.md to see if you can get the site working nice and simply.

``` bash
sudo rm -rf node_modules cache.sqlite django-error.log  # Remove files which may be owned by root
./run
```

Especially, try out the local module functionality:

``` bash
./run --module $HOME/project/vanilla-framework watch  # Build CSS dynamically, using a local version of vanilla-framework
```
